scripts/clean.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+const { promisify } = require('util');
+const rimraf = promisify(require('rimraf'));
```

Comment:
no need for rimraf. node v14 supports recursive deletes. https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fspromisesrmpath-options

Reply:
already required by cli/npm scripts for windows vs linux compatibility, but also comes with glob support built in